### DISCUSSION TITLE
What open-source alternatives are there to Obvs in .NET?

### DISCUSSION THREAD

**Discussion by SpocWeb** [christopherread/Obvs#53] [Q&A] [answered]
Hi I quite like the abstraction this libary offers, but I wonder if there are other alternatives, just to get a perspective before we make a decision.

**Comment by fadrangi** [comment, marked as answer]
MassTransit

**Reply by SpocWeb** [reply]
Yes, thank you, we are using it successfully and have also investigated into the commercial alternative nServiceBus
